pkg/kclshim/kcl.go

```diff
@@ -0,0 +1,80 @@
+package kclshim
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	krmv1 "github.com/Azure/eno/pkg/krm/functions/api/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kcl "kcl-lang.io/kcl-go"
+	"kcl-lang.io/kcl-go/pkg/spec/gpyrpc"
+)
+
+func printErr(err error) {
+	rl := krmv1.ResourceList{
+		APIVersion: krmv1.SchemeGroupVersion.String(),
+		Kind:       krmv1.ResourceListKind,
+		Items:      []*unstructured.Unstructured{},
+		Results: []*krmv1.Result{
+			{
+				Message:  err.Error(),
+				Severity: krmv1.ResultSeverityError,
+			},
+		},
+	}
+	bytes, err := json.Marshal(rl)
+	if err != nil {
+		rl.Results = append(rl.Results, &krmv1.Result{
+			Message:  fmt.Sprintf("error marshaling error response: %v", err),
+			Severity: krmv1.ResultSeverityError,
+		})
+	}
+	fmt.Print(string(bytes))
+}
+
+func Synthesize(workingDir string) {
```

Comment:
Let's get rid of the README and use godoc comments instead - on this function and maybe on the top-level package. Doesn't need to be too detailed - just something to explain how to get started with this library